### ISSUE TITLE
Object Explorer / Server Explorer Pane

### ISSUE THREAD

**Issue by alohaninja** [Microsoft/vscode-mssql#875]
Can we get support / integration with object explorer and VS Code? SSMS navigation is just awful - the SLN/Project structure leaves more to be desired. VS Code is the single tool to rule them all!

Would love to help work on this feature if possible.

**Comment by kevcunnane** [comment]
Thanks for raising this. I've added your request to #586 as it's a duplicate of this request. We're still blocked on lack of an API to support this, but it seems that VSCode may be moving to add one. If so, we can look into supporting as we've added some backend APIs that would help with this. I'm going to close out this issue since we alreay have #586 tracking this, but please feel free to monitor that issue as it has latest status etc. in it.
